### PR TITLE
Enabled DDR_Test in openJ9

### PR DESCRIPTION
**Pull request by renfeiw**
- Migrated tck_ddrext test suite into openJ9
- Removed DTFJ_TCK from the test framework
- Added core generation part into tck_ddrext.xml
- Fixed and enbaled testDDREx_General

Signed-off-by: Renfei Wang <renfeiw@ca.ibm.com>

### CODE REVIEW

**Comment by llxia** [comment]
Jenkins test all plinux,win jdk10

**Comment by llxia** [comment]
Jenkins test all win jdk10

**Comment by llxia** [comment]
Jenkins test all plinux jdk10

**Comment by llxia** [comment]
Jenkins test all plinux jdk10

**Comment by llxia** [comment]
Jenkins test all win jdk10

**Comment by llxia** [comment]
Jenkins test sanity win jdk10

**Comment by llxia** [comment]
Jenkins test all plinux jdk10

**Comment by llxia** [comment]
Jenkins test all plinux jdk10

**Comment by llxia** [comment]
Jenkins test extended zlinux,plinux,aix jdk10

**Comment by llxia** [comment]
We may want to disable DDR test compilation on aix as DDR is not supported on aix yet.

**Comment by renfeiw** [comment]
disabled compilation on aix and z/OS

**Comment by llxia** [comment]
Jenkins test extended zlinux,aix jdk10

**Comment by llxia** [comment]
Jenkins test extended zlinux,aix jdk10

**Comment by llxia** [comment]
Jenkins test sanity win jdk8

**Comment by llxia** [comment]
Jenkins test sanity win jdk8

File: test/functional/DDR_Test/build.xml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+			<fileset dir="${src}/../" includes="*.mk" />
+		</copy>
+	</target>
+
+	<target name="clean" depends="dist" description="clean up">
+		<!-- Delete the ${build} directory trees -->
+		<delete dir="${build}" />
+	</target>
+
+	<!-- Temporarily disable compilation on zos and aix until ddr is supported -->
```
Comment (llxia) [inline test/functional/DDR_Test/build.xml]: Can you add issue number?

File: test/functional/DDR_Test/src/ddrext_excludes.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  [1] https://www.gnu.org/software/classpath/license.html
+  [2] http://openjdk.java.net/legal/assembly-exception.html
+
+  SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+<excludes>	
+	<exclude name="testSym" domain="native"/>
+	<exclude name="testDclibs" domain="native"/>
+</excludes>
```
Comment (llxia) [inline test/functional/DDR_Test/src/ddrext_excludes.xml]: Please remove this file as it is not used.

**Comment by llxia** [comment]
Copyright Check failure is due to .project file. Ignore this failure.

File: test/functional/DDR_Test/build.xml (hunk 1 above)
Comment (renfeiw) [reply test/functional/DDR_Test/build.xml]: Added.

**Comment by llxia** [comment]
Jenkins test sanity win jdk10

**Comment by llxia** [comment]
Jenkins test sanity win jdk10

**Pull request merged**